src/client/client.go

```diff
@@ -63,9 +67,10 @@ func NewMetricsClientFromAddress(addr string, metrics bool, prefix string) (*API
 }
 
 // NewFromAddress constructs a new APIClient for the server at addr.
-func NewFromAddress(addr string) (*APIClient, error) {
+func NewFromAddress(addr string, maxConcurrentStreams uint) (*APIClient, error) {
```

Comment:
Good point! Updated this and added another 'sugar' constructor so less code changes